src/addon/file_server/mod.rs

```diff
@@ -224,15 +241,15 @@ impl<'a> FileServer {
         for entry in entries {
             let entry = entry.context("Unable to read entry")?;
             let metadata = entry.metadata()?;
-            let created_at = if let Ok(time) = metadata.created() {
-                format_system_date(time)
+            let date_created = if let Ok(time) = metadata.created() {
+                time.into()
             } else {
-                String::default()
+                Local::now()
             };
-            let updated_at = if let Ok(time) = metadata.modified() {
-                format_system_date(time)
+            let date_modified = if let Ok(time) = metadata.modified() {
+                time.into()
             } else {
-                String::default()
+                Local::now()
```

Comment:
I think this can be confusing for the user because instead of having an explicit text that means "We couldnt retrieve the date" we are providing the current time which would wean a file was just created which is not right.

I suggest:

- Making the dates optional for `DirectoryEntry`
- Using a fallback symbol like `-` for cases where the date wasnt found.

WDYT?

Reply:
Implemented